### ISSUE TITLE
Update download should be visible in TDV GUI

### ISSUE THREAD

**Issue by munawarb** [munawarb/Three-D-Velocity#58]
Right now, the notification of an update is done through the user's screen reader or through SAPI if no screen reader is available. This should be changed to a standard dialog so that sighted and low-vision people can tell if the game is asking to download an update or not. Further, the download progress should be displayed using a progress bar.
